File: backups/capacitor-sora-editor-1.3.13/android/sora-editor/editor/src/main/java/io/github/abc15018045126/sora/widget/snippet/variable/WorkspaceBasedSnippetVariableResolver.java

package io.github.abc15018045126.sora.widget.snippet.variable;

import androidx.annotation.NonNull;

public abstract class WorkspaceBasedSnippetVariableResolver implements ISnippetVariableResolver {

    @NonNull
    @Override
    public String[] getResolvableNames() {
        return new String[]{
                "WORKSPACE_NAME", "WORKSPACE_FOLDER"
        };
    }

}

File: capacitor-sora-editor/android/sora-editor/editor/src/main/java/io/github/abc15018045126/sora/widget/snippet/variable/WorkspaceBasedSnippetVariableResolver.kt
package io.github.abc15018045126.sora.widget.snippet.variable

abstract class WorkspaceBasedSnippetVariableResolver : ISnippetVariableResolver {

    override fun getResolvableNames(): Array<String> {
        return arrayOf(
            "WORKSPACE_NAME", "WORKSPACE_FOLDER"
        )
    }

}
